frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -342,24 +290,65 @@ const Details_ = ({ flags, obj: pvc }) => {
 const Details = connectToFlags(FLAGS.CAN_LIST_PV)(Details_);
 
 const allPhases = ['Pending', 'Bound', 'Lost'];
-const filters = [
-  {
-    filterGroupName: 'Status',
-    type: 'pvc-status',
-    reducer: (pvc) => pvc.status.phase,
-    items: _.map(allPhases, (phase) => ({
-      id: phase,
-      title: phase,
-    })),
-  },
-];
 
 export const PersistentVolumeClaimsList = (props) => {
   const Row = React.useCallback((rowProps) => <PVCTableRow {...rowProps} />, []);
+  const { t } = useTranslation();
+  const PVCTableHeader = () => {
+    return [
+      {
+        title: t('public~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('public~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+        id: 'namespace',
+      },
+      {
+        title: t('public~Status'),
+        sortField: 'status.phase',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: t('public~PersistentVolumes'),
+        sortField: 'spec.volumeName',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[3] },
+      },
+      {
+        title: t('public~Capacity'),
+        sortFunc: 'pvcStorage',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[4] },
+      },
+      {
+        title: t('public~Used'),
+        sortFunc: 'pvcUsed',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[5] },
+      },
+      {
+        sortField: 'spec.storageClassName',
+        title: t('public~StorageClasses'),
```

Comment:
I think this should be singular?

```suggestion
        title: t('public~StorageClass'),
```

Reply:
updated